.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+**JIRA Ticket:** [XXX-###](https://mfslcapstone.atlassian.net/browse/XXX-###)
+
+**Summary:**
+<!-- Please include a summary of the change, the issue it addresses, or the feature it implements. Clearly state the impact of this change on the existing system and any new functionalities being introduced. -->
+
+**Screenshots/Examples:**
+<!-- If applicable, add screenshots or examples to help explain your change. Label screenshots as necessary for clarity. -->
+
+## Type of change
+
+<!-- Please check the relevant option. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Performance enhancement (non-breaking change which improves efficiency)
+- [ ] Code cleanup (non-functional change which does not add or remove functionality)
+
+## Testing
```

Comment:
tests should be configured to be run automatically by the github actions that runs mvn package